pkg/commands/exec_live_default.go

```diff
@@ -0,0 +1,121 @@
+// +build !windows
+
+package commands
+
+import (
+	"bufio"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+	"unicode/utf8"
+
+	"github.com/kr/pty"
+	"github.com/mgutz/str"
+)
+
+// RunCommandWithOutputLiveWrapper runs a command and return every word that gets written in stdout
+// Output is a function that executes by every word that gets read by bufio
+// As return of output you need to give a string that will be written to stdin
+// NOTE: If the return data is empty it won't written anything to stdin
+// NOTE: You don't have to include a enter in the return data this function will do that for you
+func RunCommandWithOutputLiveWrapper(c *OSCommand, command string, output func(string) string) (errorMessage string, codeError error) {
+	cmdOutput := []string{}
+
+	splitCmd := str.ToArgv(command)
+	cmd := exec.Command(splitCmd[0], splitCmd[1:]...)
+
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, "LANG=en_US.utf8", "LC_ALL=en_US.UTF-8")
+
+	tty, err := pty.Start(cmd)
+
+	if err != nil {
+		return errorMessage, err
```

Comment:
does errorMessage's value ever get set in this function?

Reply:
No :), i'll set it back to just an empty string that's probably better for understating the code